src/Compiler/Driver/CompilerOptions.fs

```diff
@@ -111,7 +111,9 @@ let compilerOptionUsage (CompilerOption (s, tag, spec, _, _)) =
         else
             sprintf "%s:%s" s tag (* still being decided *)
 
-let PrintCompilerOption (CompilerOption (_s, _tag, _spec, _, help) as compilerOption) =
+let nl = Environment.NewLine
+
+let PrintCompilerOption printer (CompilerOption (_s, _tag, _spec, _, help) as compilerOption) =
```

Comment:
I think better to take a TextWriter argument rather than a printer function.  That's more standard.

Reply:
Eventually removed